sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -117,4 +124,65 @@ export class ApplicationAESTController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Triggers manual reassessment for an application.
+   * Application cannot be archived and current assessment must be in completed status.
+   * @param applicationId application id.
+   */
+  @Roles(Role.AESTAdmin)
+  @Post(":applicationId/manual-reassessment")
+  @ApiNotFoundResponse({ description: "Application id not found." })
+  @ApiUnprocessableEntityResponse({
+    description:
+      `Application current assessment expected to be '${StudentAssessmentStatus.Completed}' to allow manual reassessment or ` +
+      "application cannot have manual reassessment after being archived.",
+  })
+  async manualReassessment(
+    @Body() payload: ManualReassessmentAPIInDTO,
+    @Param("applicationId", ParseIntPipe) applicationId: number,
+    @UserToken() userToken: IUserToken,
+  ): Promise<void> {
+    try {
+      await this.studentAssessmentService.manualReassessment(
+        applicationId,
+        payload.note,
+        userToken.userId,
+      );
+    } catch (error: unknown) {
+      if (error instanceof CustomNamedError) {
+        if (error.name === APPLICATION_NOT_FOUND) {
+          throw new NotFoundException(error.message);
+        }
+        if (error.name === INVALID_OPERATION_IN_THE_CURRENT_STATUS) {
+          throw new UnprocessableEntityException(error.message);
+        }
+      }
+      throw error;
+    }
+  }
+
+  /**
+   * Gets application and assessment status details.
+   * @param applicationId application id.
+   */
+  @Get(":applicationId/application-assessment-status-details")
+  @ApiNotFoundResponse({ description: "Application id not found." })
+  async getApplicationAssessmentStatusDetails(
+    @Param("applicationId", ParseIntPipe) applicationId: number,
+  ): Promise<ApplicationAssessmentDetailsAPIOutDTO> {
+    try {
+      return await this.applicationService.getApplicationAssessmentStatusDetails(
+        applicationId,
+      );
+    } catch (error: unknown) {
+      if (
```

Comment:
The not fund check can be done in the controller once the below comment is implemented
https://github.com/bcgov/SIMS/pull/3042/files#r1541978737